### PR TITLE
docs: improve docs for Apple Silicon dylib load

### PR DESCRIPTION
**Pull request by david-allison**
Instructions for dismissing dialog with message:

```
“librsdroid-0-1-11.dylib” can’t be opened because Apple cannot check
it for malicious software.

This software needs to be updated.
Contact the developer for more information.

[Show in Finder] [OK]
```

Tested this a couple of times

Screenshot 2022-06-29 at 02 01 50

Screenshot 2022-06-29 at 03 00 15

## Learning

I love Apple more and more every day

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison** [comment]
Blocked on https://github.com/ankidroid/Anki-Android/pull/11644

**Comment by mikehardy** [comment]
Huh - took a conflict but is no longer blocked

**Review by mikehardy — APPROVED** [review]
Just needs a de-conflict then good to go

**Comment by mikehardy** [comment]
@david-allison I think I de-conflicted this in line with previous goal, taking into account refactor from Damien that we merged during last batch of backend work ?

**Comment by david-allison** [comment]
This is waiting on me to test

**Comment by divyansh-dxn** [comment]
Is there any help required on this one?

**Comment by david-allison** [comment]
It would be appreciated, thank you!

Only requires a rebase + test

**Comment by divyansh-dxn** [comment]
No the script only downloads and sets envs properly. For the first time, sill we need to grant permission.

File: AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt [hunk 1]
```diff
@@ -561,6 +561,31 @@ open class RobolectricTest : CollectionGetter {
...
+                // This software needs to be updated. Contact the developer for more information.
+                // [Show in Finder] [OK]
+                if (e.message.toString().contains("library load disallowed by system policy")) {
+                    throw IllegalStateException(
+                        """library load disallowed by system policy.
+"To fix:
+* Run the test such that the "developer cannot be verified" message appears
+* Press "OK" on the "Apple cannot check it for malicious software" prompt
+* Run the Test Again
+* Apple Menu - System Preferences - Security & Privacy - Unlock Settings - Select Allow Anyway".
```
Comment (divyansh-dxn) [inline AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt:582]: For some users, General tab may not be selected by default (usually opens with the last opened tab).

Apple Menu - System Preferences - Security & Privacy - General (tab) - Unlock Settings - Select Allow Anyway".

Comment (divyansh-dxn) [reply AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt:582]: Rest seems great ✌🏻

**Comment by david-allison** [comment]
Pending CI

File: AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt]: Thanks so much!!

**Pull request merged**